huaweicloud/services/ecs/data_source_huaweicloud_compute_instances.go

```diff
@@ -38,6 +38,10 @@ func DataSourceComputeInstances() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 			},
+			"fixed_ip_v4": {
```

Comment:
this param should be added to query params

Reply:
fixed